apps/etterlatte-saksbehandling-ui/client/src/shared/types/IDetaljertBehandling.ts

```diff
@@ -80,15 +80,17 @@ export interface IBoddEllerArbeidetUtlandet {
 
 export interface IGyldighetResultat {
   resultat: VurderingsResultat | undefined
-  vurderinger: IGyldighetproving[]
-  vurdertDato: string
+  begrunnelse: IManuellVurdering
+  vurderinger: IGyldighetproving[] //Utgår
```

Comment:
Burde ManuellVurdering heller være en del av vurderinger-listen? Slik det er nå ser det i alle fall ut fra typene som om du _må_ manuelt vurdere alt. Men det er kanskje det vi ville ende opp med? (Synes vi snakket mer som om vi kunne gå for en løsning der vi kunne overskrive i går, ikke ha noe i tillegg).

Det vil vel forsåvidt fungere hvis begrunnelse som IManuellVurdering er nullable.

Reply:
Dette var bare noe som hang igjen, dessverre. Pusher på nytt (force). Så når alt er grønt kan du jo ta en ny titt :)